src/pyscipopt/scip.pxi

```diff
@@ -5533,6 +5556,112 @@ cdef class Model:
+        up - The dual bound of the LP after branchign up on the variable
+        downvalid - Whether down stores a valid dual bound or is NULL
+        upvalid - Whether up stores a valid dual bound or is NULL
+        solval - The solution value of the variable at the last strong branching call
+        lpobjval - The LP objective value at the time of the last strong branching call"""
+
+        cdef SCIP_Real down
+        cdef SCIP_Real up
+        cdef SCIP_Real solval
+        cdef SCIP_Real lpobjval
+        cdef SCIP_Bool downvalid
+        cdef SCIP_Bool upvalid
+
+        PY_SCIP_CALL(SCIPgetVarStrongbranchLast(self._scip, var.scip_var, &down, &up, &downvalid, &upvalid, &solval, &lpobjval))
+
+        return down, up, downvalid, upvalid, solval, lpobjval
+
+    def getVarStrongBranchLastNode(self, Variable var):
+        """Get the node number from the last time strong branching was called on the variable"""
+
+        cdef SCIP_Longint node_num
+        node_num = SCIPgetVarStrongbranchNode(self._scip, var.scip_var)
+
+        return node_num
+
+    def strongBranchVar(self, Variable var, itlim, idempotent=False, integral=False):
+        """ Strong branches and gets information on column variable.
+
+        :param Variable var: Variable to get strong branching information on
+        :param itlim: LP iteration limit for total strong branching calls
+        :param idempotent: Should SCIP's state remain the same after the call?
+        :param integral: Boolean on whether the variable is currently integer.
+        """
+
+        cdef SCIP_Real down
+        cdef SCIP_Real up
+        cdef SCIP_Bool downvalid
+        cdef SCIP_Bool upvalid
+        cdef SCIP_Bool downinf
+        cdef SCIP_Bool upinf
+        cdef SCIP_Bool downconflict
+        cdef SCIP_Bool upconflict
+        cdef SCIP_Bool lperror
+
+        if integral:
+            PY_SCIP_CALL(SCIPgetVarStrongbranchInt(self._scip, var.scip_var, itlim, idempotent, &down, &up, &downvalid, 
+                                                   &upvalid, &downinf, &upinf, &downconflict, &upconflict, &lperror))
+        else:
+            PY_SCIP_CALL(SCIPgetVarStrongbranchFrac(self._scip, var.scip_var, itlim, idempotent, &down, &up, &downvalid, 
+                                                    &upvalid, &downinf, &upinf, &downconflict, &upconflict, &lperror))
+
+        return down, up, downvalid, upvalid, downinf, upinf, downconflict, upconflict, lperror
+
+    def updateVarPseudoCost(self, Variable var, valdelta, objdelta, weight):
+        """Updates the pseudo costs of the given variable and the global pseudo costs after a change of valdelta
+         in the variable's solution value and resulting change of objdelta in the LP's objective value.
+         Update is ignored if objdelts is infinite. Weight is in range (0, 1], and affects how it updates
+         the global weighted sum.
+
+         :param Variable var: Variable whos psuedo cost will be updated
```

Comment:
psuedo -> pseudo

I think we're snake case for python variables (so val_delta and obj_delta?). I personally don't mind it staying like this

Reply:
Changed.

For the snake case, I am a big fan for Python variables. I dislike any other case. I feel like this was never a standard previously though, and all Cython variables are the exact opposite.